doc/parameters.rst.jinja

```diff
@@ -0,0 +1,39 @@
+##################
+List of Parameters
+##################
+
+
+The following is the full list of parameters and their values as used in EOS v{{ version }}.
+
+{% for section in sections %}
+
+{{ "*" * len(section.name) }}
+{{ section.name }}
+{{ "*" * len(section.name) }}
+
+{% for group in section.groups %}
+{{ group.name }}
+{{ "=" * len(group.name) }}
+
+.. list-table::
+   :widths: 40, 20, 20, 20
+   :header-rows: 1
+
+   * - Qualified Name
+     - Representation
+     - Default Value
+     - Unit
+
+   {% for param in group.parameters %}
+       .. _{{ param.link_key }}:
+   * - ``{{ param.qualified_name }}``
+     - {{ param.latex }}
+     - {{ param.value }}
+     - {{ param.unit }}
+
+   {% endfor %}
+
+{{ group.description }}
```

Comment:
The group description sometimes includes display math (equation, align). I remove this for now, since the implementation is not trivial.

Reply:
We can translate that as well, can we not?

Reply:
After trying for a couple minutes, I ran into trouble with align, for example, here:

https://github.com/eos/eos/blob/43e942742fb0946a6885fa9c49bb6ea66367c8f0/eos/parameters/sm-and-eft.yaml#L880-L892

Reply:
Isn't the only thing needed here to replace
```
\begin{align}
   %content
\end{align}
```
with
```
.. math::

   %content
```
or am I missing something?

Reply:
See also here: https://sphinx-rtd-trial.readthedocs.io/en/latest/ext/math.html#directive-math

Reply:
It should work now!
There was a minor problem with newlines in the yaml files and aligned equations with only a single line (and a single `&` character).